### DISCUSSION TITLE
Vaultwarden web removes checked items, when closing window

### DISCUSSION THREAD

**Discussion by ramlev** [dani-garcia/vaultwarden#4062] [Q&A]
When using Vaultwarden web (newest - and all earlier versions) and want to delete a list of items from your vault.

1. go to the vault list.
2. mark checkboxes on one or several items
3. accidentally click outside a check, so an edit item popup is visible.
4. press the 'x' or 'Cancel'
5. All the items you have selected, is now unselected, and you can start over again.
6.

**Comment by BlackDex** [comment]
These specific issues have nothing to do with Vaultwarden and are specific to the web-vault client.

Not something we can fix on the server backend side.
